src/compiler/scala/tools/nsc/symtab/SymbolLoaders.scala

```diff
@@ -329,10 +297,7 @@ abstract class SymbolLoaders {
 
       val loaders = SymbolLoaders.this.asInstanceOf[SymbolLoadersRefined]
 
-      override def classFileLookup: util.ClassFileLookup[AbstractFile] = settings.YclasspathImpl.value match {
-        case ClassPathRepresentationType.Recursive => platform.classPath
-        case ClassPathRepresentationType.Flat => platform.flatClassPath
-      }
+      override def classFileLookup: ClassPath = platform.classPath
```

Comment:
All the `classFileLookup` methods should be reviewed and either removed or renamed to classpath. 

Here's the commit that introduced them for a guide: 74483771dcdfc963b28325d8ea3698df273370a1